src/AppBanner/index.tsx

```diff
@@ -32,18 +28,14 @@ export type BrazeMessageProps = {
 import type { ButtonTheme } from '@guardian/source-react-components';
 
 export type Props = {
-    logButtonClickWithBraze: BrazeClickHandler;
-    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
-    ophanComponentId?: string;
```

Comment:
As with `BannerWithLink` I don't think we want to lose the `ophanComponentId` arg here. I _think_ this will also remove the need to cast with `as string` below.